src/vivarium_nih_us_cvd/components/causes.py

```diff
@@ -0,0 +1,163 @@
+from typing import Any, Callable, Dict, List, Union
+
+import pandas as pd
+from vivarium import ConfigTree
+from vivarium.framework.engine import Builder
+from vivarium_public_health.disease import (
+    BaseDiseaseState,
+    DiseaseModel,
+    DiseaseState,
+    RecoveredState,
+    SusceptibleState,
+)
+from vivarium_public_health.utilities import TargetString
+
+from vivarium_nih_us_cvd.constants.paths import CAUSE_RISK_CONFIG
+
+
+class Causes:
+    # todo could modify the component configuration parser plugin
+    def __init__(self):
+        self._sub_components = self.get_disease_models()
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def name(self) -> str:
+        return "causes"
+
+    @property
+    def sub_components(self) -> List:
+        return self._sub_components
+
+    ##################
+    # Helper methods #
+    ##################
+
+    def get_disease_models(self) -> List:
+        disease_models = []
+
+        full_config = ConfigTree(layers=["default", "model_spec"])
+        full_config.update(CAUSE_RISK_CONFIG, layer="model_spec")
+        self.add_default_config_layer(full_config)
+        config = full_config.causes
+
+        for cause_name, cause_config in config.items():
+            states: Dict[str, BaseDiseaseState] = {

```

Comment:
Jeez, we add typing outside of functions so rarely that I had to stare at this for a second

Reply:
I'm almost certain I can find code one or both of you've written with this lol.